.github/workflows/algolia/src/syncAlgoliaIndex.ts

```diff
@@ -0,0 +1,114 @@
+import fs from "fs";
+import path from "path";
+import matter from "gray-matter";
+import algoliasearch from "algoliasearch";
+
+type DocsFrontMatter = {
+  content: string;
+  data: {
+    objectId: string;
+    title?: string;
+    description?: string;
+    slug?: string;
+    tags?: Array<string>;
+  };
+};
+
+const getAllFiles = function (
```

Comment:
Looking at how we write functions elsewhere, we can probably omit `function` keyword and go for `() => {}` instead. This can help us in the future when we enable shared ESLint rules in this repo.